packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -437,8 +439,23 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
       child: TextFieldTapRegion(
         child: AutocompleteHighlightedOption(
           highlightIndexNotifier: _highlightedOptionIndex,
-          child: Builder(
-            builder: (BuildContext context) => widget.optionsViewBuilder(context, _select, _options),
+          child: UnconstrainedBox(
```

Comment:
Why the UnconstrainedBox? The overlay should be large enough to accommodate the menu I think?

Reply:
Without that the width of the SizedBox is not respected and the options are always full width. I think being in an Overlay gives it weird constraints.

Reply:
That doesn't seem to work because of the CompositedTransformFollower widget. Without that Positioned does work like you said.

Reply:
`Positioned` has to be placed as the root of the overlay child. I think the problem is the overlay is giving the overlay child tight constraints.

Reply:
Ah that worked, thanks.

Reply:
Actually, with Positioned it was completely unconstrained in both directions, which caused layout errors in the options builder. I changed it to use ConstraintsTransformBox to just loosen the constraints in both directions, what do you think of that?

Reply:
Talked offline: I should try setting the width at the Positioned widget instead of below in a SizedBox.